### ISSUE TITLE
[ENHANCEMENT] Mute "Double click tray icon" notification

### ISSUE THREAD

**Issue by godfilma** [Davidobot/BetterJoy#665]
On Windows 10, every time the program is opened, and every time the program is minimized I get a notification that I can double click the tray icon to re-open the program window. I'm tired of getting the notification, but I don't want to mute the program from sending notifications because I want an alert if my controller battery is low.

I would like an additional option to prevent the sending of that specific notification, but to still receive the battery alert.

Thank you very much, this program rocks!

**Comment by eqagunn** [comment]
Yes, please! Trivial, but very annoying. If I recall correctly, in case of the other apps, such message would stop appearing once dismissed by clicking onto it. The message would inform you to do that as well. On the other hand, an option in the configuration file would do as well.
